sql/sql_table.cc

```diff
@@ -8106,6 +8106,14 @@ static bool create_table_impl(
       }
       create_info->db_type = engine_type;
     }
+    if (alter_info->has_compressed_columns() &&
```

Comment:
Is it possible to do this check in native Partitioning code? (see sql/partitioning/partition_base.cc) Partition_base::create() ?

Reply:
Ok, better to leave validation to individual SE's